ms365_accountcreator/__init__.py

```diff
@@ -34,6 +36,8 @@
     value = environ.get(env_var, APP.config.get(env_var))
     if value is None:
         pass
+    elif isinstance(value, Number):
+        pass
```

Comment:
You are right, this might be a problem, if the env var is not set. But we have the same problem with Boolean.
I'd suggest this:
 ```suggestion
    value = environ.get(env_var, None)
    if value is None:
        continue
```